src/base/print/printRxnFormula.m

```diff
@@ -11,14 +11,17 @@
 % INPUTS:
 %    model:             COBRA model structure
 %
-% OPTIONAL INPUTS:
+% OPTIONAL INPUTS
 %    rxnAbbrList:       Abbrs of reactions whose formulas are to be printed
+%
+% OPTIONAL INPUTS as 'ParameterName',Value pairs:
```

Comment:
This will break the documentation generation ... The header can only be `OPTIONAL INPUTS:`